README.md

```diff
@@ -216,6 +216,32 @@ This command does an `npm publish` and sets npm tags based on the repository sta
 
 Note: it appears that npm also tags the very first published build of each repo with "latest". 
 
+### status
```

Comment:
It might be good to give an example of the JSON output here maybe?

Reply:
Yep. Done.